product-service/serverless.yml

```diff
@@ -40,3 +84,10 @@ functions:
           method: post 
           cors: true
 
+  catalogBatchProcess:
+    handler: handler.catalogBatchProcess
+    events:
+      - sqs:
+          batchSize: 1
```

Comment:
should be 5 according to task description
![image](https://user-images.githubusercontent.com/16606092/100064192-ddda8a80-2e3a-11eb-8614-a8be1627f374.png)

Reply:
updated to 3 (dont want to create csv with a lot of data)